CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# CONTRIBUTING
+Contributions to grpc-json-client are welcome by all!
+## Build Environment Setup
+### Dependencies
+1. Install [CMake](https://cmake.org/) and add it to `path`.
+2. Install [gRPC](https://github.com/grpc/grpc) by following the instructions [here](https://github.com/grpc/grpc/blob/master/BUILDING.md).
+### Getting the Source
+1. Clone the repo:
+```
+> git clone https://github.com/ni/grpc-json-client.git
+> cd grpc-json-client
+```
+2. Initialize submodules:
+```
+> git submodule update --init
+```
+### Building on Windows
+1. Create a build directory:
+```
+> mkdir build
+> cd build
+```
+2. Generate CMake cache:
+```
+> cmake .. "-DCMAKE_PREFIX_PATH=C:\Program Files (x86)\grpc"
```

Comment:
Do you know if/why grpc is required in `third_party` when you're setting it up here? Is it a difference of linkage deps vs exe deps (i.e., protoc)?

Reply:
The original reason was we needed a proto file from the grpc repo to generate a reflection client. It was a little overkill for one file at the time.  Now with the added option of building gRPC as a submodule it makes more sense to have grpc in third_party since we need the reflection files anyway.